test/run-tests

```diff
@@ -829,14 +869,26 @@ def handle_task(gh, args, config, task, ansible_id, dry_run=False):
         sys.exit(1)
 
 
-def check_environment():
+def check_environment(args):
     """
     Check whether the environment is sane.
     Intent here is to fail early for example if /dev/kvm is not available.
     """
-    if not os.access("/dev/kvm", os.R_OK | os.W_OK):
-        logging.critical("test-harness needs access to /dev/kvm, aborting")
-        sys.exit(1)
+    if args.backend == KVM_BACKEND:
+        if not os.access("/dev/kvm", os.R_OK | os.W_OK):
+            logging.critical("test-harness needs access to /dev/kvm, aborting")
+            sys.exit(1)
+
+    elif args.backend == CITOOL_BACKEND:
+        if not os.path.isfile(os.path.join(args.config, "config-citool.json")):
+            logging.critical("test-harness needs config-citool.json file, aborting")
+            sys.exit(1)
+
+        try:
+            run("citool" "--help")
```

Comment:
missing comma between `"citool"` and `"--help"`

Reply:
With the comma, this test also fails - `gluetool` is not in the PATH - if I add
`os.environ["PATH"] = "/opt/gluetool/bin:" + os.environ["PATH"]` then citool works

Reply:
`run(CITOOL_COMMAND, "--help")`

Reply:
Fixed